app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -206,10 +207,12 @@ class DefaultBrowserToolbarController(
                 activity.components.analytics.metrics.track(Event.FindInPageOpened)
             }
             ToolbarMenu.Item.ReportIssue -> {
-                val currentUrl = currentSession?.url
-                currentUrl?.apply {
-                    val reportUrl = String.format(BrowserFragment.REPORT_SITE_ISSUE_URL, this)
-                    activity.components.useCases.tabsUseCases.addTab.invoke(reportUrl)
+                val selectedTab = activity.components.core.store.state.selectedTab
+                selectedTab?.let {
+                    val currentUrl = it.content.url
+                    val reportUrl = String.format(BrowserFragment.REPORT_SITE_ISSUE_URL, currentUrl)
+                    val private = it.content.private
```

Comment:
Could we get this info from the `BrowsingModeManager` that we're already passing in to the `BrowserToolbarController`?

Reply:
IMO this is a more reliable way to check the private mode.